### ISSUE TITLE
What kind of information should represent on home page

### ISSUE THREAD

**Issue by isuriupeksha** [systers/language-translation#61]
@rosariorobinson Currently home page has only user profile information. But I suggest to represent most useful information once user login to the system. So we can show `published` articles on home page. 

Then `Article` section contains all article whether they are `draft`, `published` and `reviewing` for admin. Contributes can view only their posted articles. 

Welcome your feedback!

**Comment by wonook** [comment]
I agree on that the root page should include some other useful information rather than the user's information. We can have some discussion about what to put on the root page on this thread.

**Comment by apoorvasingh17** [comment]
Hi, can I take this issue up?
